code/game/area/Rift_Areas.dm

```diff
@@ -1,3 +1,125 @@
+//Debug areas
+/area/rift/surfacebase
+	name = "Rift Debug Surface"
+
+/area/rift/transit
+	name = "Rift Debug Transit"
+	requires_power = 0
+
+/area/rift/space
+	name = "Rift Debug Space"
+	requires_power = 0
+
+// Elevator areas.
+/area/turbolift/runder/level2
+	name = "under (level 2)"
+	lift_floor_label = "Underground 2"
+	lift_floor_name = "Atmospherics, Chapel, Mining, Bunker"
+	lift_announce_str = "Arriving at underground level two."
```

Comment:
Remind me to make a macro to create turbolift "floors"
It'd be something like:

```
/// Example macro params
CREATE_ELEVATOR_FLOORS(ELEVATOR_ID, FLOOR_COUNT, LABEL_PREFIX, list/FLOOR_DETAILS)
```

```
/// Actual Example:
CREATE_ELEVATOR_FLOORS(runder, 2, "Underground", list/FLOOR_DETAILS( \
	1 = "Engineering", \
	2 = "Atmospherics, Chapel, Mining, Bunker", \
)
```
🤷